pkg/convert/arb.go

```diff
@@ -141,11 +142,32 @@ func ToArb(translation model.Translation) ([]byte, error) {
 	// dst length = number of messages + number of potential descriptions (same as number of messages) + locale.
 	dst := make(map[string]interface{}, len(translation.Messages)*2+1)
 
+	patternsToMsg := func(patterns []ast.Pattern) string {
```

Comment:
i think this will be also used in other converts, so might be good idea to move it outside this function